package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "suomifi-ui-components",
-  "version": "7.0.0-beta.1",
+  "version": "7.0.0-beta.2",
```

Comment:
Normally we wouldn't want to change the version during "normal" PRs.
Separate PR for increasing version number.

But this might be just some mishap when merging/rebasing and does not cause any real problem.

Reply:
Yes, this is a bit weird since it's a change made in a previous PR, not this one